Common/Basic comppenants/CartTable.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OpenQA.Selenium;
+
+namespace Infrastructure
+{
+    public class CartTable : ComppenantBase
+    {
+        public List<IWebElement> Garbage { get; set; }
+        public List<IWebElement> Price { get; set; }
+        public List<IWebElement> NumOfItems { get; set; }
+        private IWebElement Quantity { get; set; }
+
+
+        public CartTable(IWebDriver driver, IWebElement parentElement) : base(driver, parentElement)
+        {
+            try
+            {
+                Garbage = ParentElement.FindElements(By.CssSelector(".icon-trash")).ToList();
+                Price = ParentElement.FindElements(By.CssSelector(".cart_total .price")).ToList();
+                NumOfItems = ParentElement.FindElements(By.CssSelector(".cart_quantity.text-center")).ToList();
+                Quantity = ParentElement.FindElement(By.CssSelector("#summary_products_quantity"));
+            }
+
+            catch
+            {
+                Quantity = ParentElement.FindElement(By.CssSelector(".alert.alert-warning"));
+            }
+        }
+
+        public int GetQuantity()
```

Comment:
you should consider refactoring this function. you have a lot of conversions that you could probably get rid of.